src/imageops/colorops.rs

```diff
@@ -104,21 +96,17 @@ where
 
     let percent = ((100.0 + contrast) / 100.0).powi(2);
 
-    // TODO use pixels_mut?
-    for y in 0..height {
-        for x in 0..width {
-            let f = image.get_pixel(x, y).map(|b| {
-                let c: f32 = NumCast::from(b).unwrap();
-
-                let d = ((c / max - 0.5) * percent + 0.5) * max;
-                let e = clamp(d, 0.0, max);
+    image.pixels().map(|(x, y, pixel)| {
```

Comment:
Doesn't this need to be a `for` loop rather than a call to `.map()`?

Reply:
in the past code, it's was a map so I kept it but I can use a loop if you want.

Reply:
The past code made use of `Pixel::map`, not `Iterator::map`.

Reply:
Oops. I will correct that